UnitTests/MeshTests.h

```diff
@@ -125,50 +128,80 @@ DEFINE_TEST_CASE(MeshWriteBinary)
         ), "The written mesh is wrong!");
 }
 
+
+//function for copying files
+inline void copyFile(const std::string from, const std::string to)
+{
+    std::ifstream src(from, std::ios::binary);
+    std::ofstream dest(to, std::ios::binary);
+    dest << src.rdbuf();
+};
+
+
 DEFINE_TEST_CASE(FoundationNormals)
```

Comment:
Please move all of those test cases to ApplicationTests. So that they will be in one place.

Reply:
Okey